clusters/production/kustomization.yaml

```diff
@@ -4,6 +4,7 @@ resources:
 - flux-system
 - ./imageUpdateAutomation.yaml
 - ../../components/third-party/blob-csi-driver
+- ../../components/third-party/csi-secrets-store-provider-azure
```

Comment:
should this be in the development overlay instead?

Reply:
Yes - this is a mistake. Fixed.